lvgl/src/display.rs

```diff
@@ -1,5 +1,100 @@
 use crate::objx::ObjectX;
 use core::ptr;
+use embedded_graphics;
+use embedded_graphics::prelude::*;
+use embedded_graphics::{drawable, DrawTarget};
+use core::mem::MaybeUninit;
+use embedded_graphics::pixelcolor::Rgb888;
+use core::marker::PhantomData;
+
+
+pub struct DisplayDriver<'a, T>
+    where
+        T: DrawTarget<Rgb888>,
```

Comment:
Would be nice to have this type more generic. Not all embedded systems will support that.